components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesEndpoint.java

```diff
@@ -76,6 +78,19 @@ public void startRoute(String id) throws Exception {
         camelContext.getRouteController().startRoute(id);
     }
 
+    public String resetRoute(String id) throws Exception {
+        ManagedRouteMBean resetRoute = camelContext.getManagedRoute(id, ManagedRouteMBean.class);
+        if (resetRoute != null) {
+            resetRoute.reset(true);
+        } else {
+            String description = "The provided route id does not exists";
+            ResponseEntity<?> objectResponseEntity = ResponseEntity.status(HttpStatus.NO_CONTENT).body(description);
+            ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
All right. No issues. Thanks you for again reviewing this. I was waiting to get a final ack on the status code on what needs to be returned  as some of the reviewers also mentioned 204.  Now that it is clarified, I will make the changes. 
@davsclaus, i returned JSON because, throughout the route controller example, we have the output as json message for displaying the stats.